docs/SCaC_Documentation.md

```diff
@@ -97,7 +102,7 @@ This document defines all necessary prerequisites for the implementation of Azur
 
 * **Deploy to Azure** 
   * Add your policies, initiatives and assignments to their respective folders and make changes as needed 
-    * NOTE: All policies must keep the file name azurepolicy.json and be in their own sub folder under the policies folder 
+  * 
```

Comment:
@nicko3185 looks like this is an extra bullet point with not content. Is this safe to remove?